lib/init/build.zig.zon

```diff
@@ -5,8 +5,6 @@
     .version = "0.0.0",
 
     // This field is optional.
-    // This is currently advisory only; Zig does not yet do anything
-    // with this value.
```

Comment:
For completeness you could also remove the corresponding sentence in https://github.com/ziglang/zig/blob/master/doc/build.zig.zon.md

Reply:
True! Thanks for noticing.